clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -118,6 +104,31 @@ def __init__(
                 metalake_name=metalake_name,
                 auth_data_provider=SimpleAuthProvider(),
             )
+        elif auth_type == GVFSConfig.OAUTH2_AUTH_TYPE:
```

Comment:
The code above should be `auth_type == GVFSConfig.SIMPLE_AUTH_TYPE`?

Reply:
Done